packages/core/email/server/src/types.ts

```diff
@@ -0,0 +1,40 @@
+export interface EmailConfig {
+  provider: string;
+  providerOptions?: object;
+  settings?: {
+    defaultFrom?: string;
+  };
+}
+
+export interface StrapiConfig {
```

Comment:
Can you extend LoadedPlugin.config for this, since that's ultimately what it is?

You might need to update from main